seda/chain.json

```diff
@@ -0,0 +1,185 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "seda",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://www.seda.xyz/",
+  "pretty_name": "SEDA",
+  "chain_id": "seda-1",
+  "bech32_prefix": "seda",
```

Comment:
Please register this prefix
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
Opened a PR here https://github.com/satoshilabs/slips/pull/1720

Reply:
merged.

Reply:
thanks @hacheigriega !